src/js/chartAuthoring.js

```diff
@@ -237,6 +360,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onUpdateDataEntryPanel.fire();
```

Comment:
@cindyli - I would be concerned that would imply the event is fired whenever the data entry panel is updated, such as by user action; in this case it's actually fired when the floe.chartAuthoring.updateDataEntryPanel function that updates the data entry panel and the corresponding pie chart from a dataset completes.

That said, I think that function itself is probably badly named - propose renaming it to "updateInterfaceFromDataSet", and will rename the event accordingly. Does that sound workable?